gremlin-core/pom.xml

```diff
@@ -113,6 +113,12 @@ limitations under the License.
             <artifactId>compile-testing</artifactId>
             <version>0.10</version>
             <scope>test</scope>
+            <exclusions>
```

Comment:
could you please explain why this exclusion was necessary?

Reply:
com.google.testing.compile imposes a transitive dependency from JDK's tools.jar. It is not published in the Maven central. If it is not clear if tools.jar is used at all - the unit and integration tests pass with tools.jar excluded.
The import of tinkerpop-core as a Maven project in Eclipse registers tools.jar as a library, but Eclipse m2e cannot resolve it. This blocks the project's compilation in the IDE.